crypto/evp/evp_rand.c

```diff
@@ -431,15 +431,31 @@ const OSSL_PARAM *EVP_RAND_gettable_ctx_params(const EVP_RAND *rand)
     if (rand->gettable_ctx_params == NULL)
         return NULL;
     return rand->gettable_ctx_params(
-               ossl_provider_ctx(EVP_RAND_provider(rand)));
+               NULL, ossl_provider_ctx(EVP_RAND_provider(rand)));
```

Comment:
you missed the ones in this file :) Dont hate me..

Reply:
:(